src/Inventory.zig

```diff
@@ -304,21 +313,37 @@ pub const Sync = struct { // MARK: Sync
 			executeCommand(payload, source);
 		}
 
+		pub fn createManagedInventory(len: usize, typ: Inventory.Type, source: Source, zon: ZonElement) u32 {
```

Comment:
Again, overspecification, isn't that a method on struct called `Inventory`? How many other create methods do you have here? IMO important information here is that this method creates and that type of inventory is **externally** managed, so `createExternallyManaged`